manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -84,6 +84,12 @@ spec:
             seLinuxOptions:
               level: "s0"
           env:
+          {{- if .RuntimeSpec.UseDtk }}
+            - name: DTK_OCP_DRIVER_BUILD
+              value: "true"
+            - name: DTK_OCP_NIC_SHARED_DIR
```

Comment:
nit: "NIC" doesnt add much meaning.

but maybe the ship has sailed...

Reply:
Yeah, it is in sync with MOFED container code